pyfarm/jobtypes/core/internals.py

```diff
@@ -292,46 +292,6 @@ class Process(object):
     def __init__(self):
         self.start_called = False
         self.stop_called = False
-        self._stopped_deferred = None
-        self._started_deferred = None
-
-    @property
-    def stopped_deferred(self):
```

Comment:
Since Process._start() will now register the tasklog before actually starting the process, it will have to return these two properties before the process has actually started. So, yeah, in this case we will have to access both of these before self.start_called is set to True.